test/ibmq/test_basic_server_paths.py

```diff
@@ -36,6 +42,17 @@ def setUpClass(cls, providers):
         super().setUpClass()
         cls.providers = providers  # Dict[str, AccountProvider]
         cls.last_week = datetime.now() - timedelta(days=7)
+        # Setup list of jobs to check have been deleted.
+        cls.jobs_to_delete: List[Tuple[AccountProvider, str]] = []
+
+    def tearDown(self) -> None:
+        # Ensure all jobs are deleted.
+        for provider, job_id in self.jobs_to_delete:
+            try:
+                provider.runtime.delete_job(job_id)
+            except RuntimeJobNotFound:
+                pass
+        return super().tearDown()
```

Comment:
return is not necessary here. Just `super().tearDown()` should be good.

Reply:
Updated!